src/guide/making-a-change.md

```diff
@@ -68,6 +68,12 @@ If you create a new file, copy the license text from a nearby file. Do not add c
 
 If you move code from an existing file to a new file, bring the entire copyright + license header over to the new file.
 
+## Changes Late in the Release
+
+You should always aim to get your changes integrated as early in a release as possible to maximize the amount of testing done before release. If you risk running late it's almost always a better idea to defer the change to the next release rather than trying to squeeze it in in the last minute. The next release is only six months out, the world can wait even though you are eager.
+
+Integrating during the rampdown phase is not recommended. The bar to integrate a change in the rampdown phase is considerably higher than during the normal development phase. See [The JDK Release Process] for more information. Also note that changes that are made late in the release shouldn't expect localization since this in itself takes some time.
```

Comment:
```suggestion
Integrating during the rampdown phase is not recommended. The bar to integrate a change in the rampdown phase is considerably higher than during the normal development phase. See [The JDK Release Process] for more information. Also note changes that are made late in the release shouldn't expect localization since this in itself takes some time.
```

Is the last sentence easier to read and understand if you omit the relative conjunction ‘that’?

Reply:
I don't think so. Your suggestion would require a comma after "note", and I don't think that sentence would be easier to read. It could be simplified by removing "that are" (the second that) instead.

Reply:
> I don't think so. Your suggestion would require a comma after "note", and I don't think that sentence would be easier to read.

You're right. The sentence is easier to understand with both “that” on their own places.

> It could be simplified by removing "that are" (the second that) instead.

No, the second “that” can't be removed because it's the subject of the dependent sentence “that are made late in the release” which describes _changes_.